contrib/jenkins_tests/style.sh

```diff
@@ -39,18 +40,29 @@ for file in $check_files; do
     ret=$((1-$?))
     nerrors=$((nerrors+ret))
     set -eE
-    i=$((i+1))
 
     file=$(basename ${file})
     if [ $ret -gt 0 ]; then
+        i=$((i+1))
         echo "not ok $i $file # See: ${file}.diff" >> $style_tap
     else
-        echo "ok $i $file" >> $style_tap
+        rm -rf ${style_diff}
     fi
 done
+if [ $nerrors -eq 0 ]; then
+    echo "1..1" > $style_tap
+    echo "ok 1 all $(echo "$check_files" | wc -l) files" >> $style_tap
```

Comment:
I think you want output text to say "ok all X files" and not "ok 1 all X files"?

Reply:
you are right

Reply:
It is `tap` format. See http://testanything.org/

Reply:
Ah, gotcha. 

Minor suggestion: Adding a hyphen before string text would improve readability. This is done in the example in the link you shared (`ok 3 - Read the rest of the file`).